cms/envs/common.py

```diff
@@ -41,12 +41,8 @@
 from xmodule.modulestore.edit_info import EditInfoMixin
 
 ############################ FEATURE CONFIGURATION #############################
+STUDIO_NAME = "Studio"
```

Comment:
We've decided to leave this as "Studio" because with small exception, we've totally dropped "cms". In fact, `paver devstack cms` doesn't even _work_ as a command (no matter how many freaking times I type it in). On the mailing list and in documentation, we refer to "studio" almost exclusively. I think calling this "CMS_NAME" would be confusing to everyone, both users and people coding (would you remember to use CMS NAME? I think we're more likely to remember STUDIO NAME).